server/apps/cassandra-app/src/assemble/license-for-binary.txt

```diff
@@ -1103,37 +1103,3 @@ Within the lib directory
             implementations of the Specification to develop, distribute
             and use a Compliant Implementation.
             
-
-    licensed under the BSD (3-clause style) http://jetm.void.fm/license.html  (as follows)
-
-            Copyright (c) 2004, 2005, 2006, 2007 void.fm
-            All rights reserved.
-
-            Redistribution and use in source and binary forms, with or without modification,
-            are permitted provided that the following conditions are met:
-
-            * Redistributions of source code must retain the above copyright notice, this list
-              of conditions and the following disclaimer.
-
-            * Redistributions in binary form must reproduce the above copyright notice, this
-              list of conditions and the following disclaimer in the documentation and/or
-              other materials provided with the distribution.
-
-            * Neither the name void.fm nor the names of its contributors may be
-              used to endorse or promote products derived from this software without specific
-              prior written permission.
-
-            THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
-            ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
-            WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
-            IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
-            INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
-            BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
-            DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
-            LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
-            OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
-            OF THE POSSIBILITY OF SUCH DAMAGE.
-
-        from JETM  http://jetm.void.fm
-            jetm-1.2.3.jar
-            jetm-optional-1.2.3.jar
```

Comment:
We are no longer using this, correct?

Reply:
My understanding is that it was removed a long time ago in 91848703279fa96cd1ab0b202b63554655e02fe7, and I suspect the files changed in this PR were overlooked.